packages/buckets/src/buckets.ts

```diff
@@ -4,11 +4,18 @@ import { API, APIPushPath } from '@textile/buckets-grpc/buckets_pb_service'
 import CID from 'cids'
 import { Channel } from 'queueable'
 import { grpc } from '@improbable-eng/grpc-web'
-import { Context } from '@textile/context'
+import { Context as Ctx } from '@textile/context'
```

Comment:
So here we still import Context lib, but alias it to Ctx because we're only using it to create a default Context. I'm calling this a 'minimal' fix because, in typescript, if we reference buckets.context, it'll only satisfy the minimal Context interface defined here, which isn't going to work _outside_ Buckets (it doesn't have all the with* methods. Ideally, we'd export the full interface from @textile/context and use that here, but to keep things simple an to avoid having to make those changes right away, we can use this minimal fix.